pkg/portercontext/context.go

```diff
@@ -443,7 +442,7 @@ func (c *Context) CopyDirectory(srcDir, destDir string, includeBaseDir bool) err
 
 	return c.FileSystem.Walk(srcDir, func(path string, info os.FileInfo, err error) error {
 		if err != nil {
-			return errors.WithStack(err)
+			return fmt.Errorf("%w", err)
```

Comment:
Let's just return err here. I'm not quite sure what I was going for with the WithStack.
```suggestion
			return err
```